joomla/joomla-cms
# [5.0] Improve CSS/JS build times

### Summary of Changes

Replace Terser and PostCSS with ESBuild (Go) and LightningCSS (Rust) to improve build times.

### Testing Instructions

1. Run `npm i`
2. Perform a sanity test to ensure there and no JS errors or styling issues

### BEFORE (local machine)

> Timer: Build finished in **35396 ms**

### AFTER (local machine)

> Timer: Build finished in **17231 ms**

@dgrammatiko I haven't removed PostCSS completely because the build scripts are still using your custom PostCSS plugin, which I didn't want to touch.
